### ISSUE TITLE
iOS 12.2 Scrolling issues

### ISSUE THREAD

**Issue by asa47** [ionic-team/ionic-v1#400]
#### Short description of the problem:
There is a bug in iOS 12.2 that breaks the scrolling on the device, as if the scrollable area becomes frozen. This scroll freeze persists until some other change is made to the DOM.

Any suggestion on what we should do to fix the issue on ionic1 applications?

#### What behavior are you expecting?
Scrolling should work on scrollable area. 

**Other information:** 
The issue has been fixed in ionic-angular 3.9.5 (ionic-v3)
 [Patch for iOS 12.2 scrolling bug on ionic-angular 3.9.5](https://github.com/ionic-team/ionic-v3/commit/cf65fad)

**Which Ionic Version?** 1.x

**Run `ionic info` from terminal/cmd prompt:** (paste output below)

```
Ionic:

   ionic (Ionic CLI) : 4.12.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework   : ionic1 1.2.1
   @ionic/v1-toolkit : 1.0.12

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : ios 4.4.0
   Cordova Plugins       : cordova-plugin-ionic-webview 2.3.2, (and 16 other plugins)

System:

   ios-deploy : 1.9.2
   NodeJS     : v11.1.0 (/usr/local/bin/node)
   npm        : 6.4.1
   OS         : macOS Mojave
   Xcode      : Xcode 10.1 Build version 10B61
```

**Comment by ronaldso55** [comment]
Facing the same issue here.  Did you manage to find a workaround @asa47 ?

**Comment by asa47** [comment]
@ronaldso55 Yes, inside your ionic.bundle.js, look for the `SlideDrag.prototype.end` function, and modify the `ionic.requestAnimationFrame` callback method with the following:

```
ionic.requestAnimationFrame(function() {
        if (restingPoint === 0) {

          // Fix iOS 12.2 scrolling issue
          if(!(self && self._currentDrag && self._currentDrag.content)){ 
            return;
          }

          self._currentDrag.content.style[ionic.CSS.TRANSFORM] = '';
          var buttons = self._currentDrag.buttons;
.
.
.
```

**Comment by ronaldso55** [comment]
Thanks @asa47!  It worked like a charm.  Much appreciated.
